tests/integration/cmd_devfile_build_images_test.go

```diff
@@ -168,7 +168,7 @@ var _ = Describe("odo devfile build-images command tests", func() {
 						containerImage := "localhost:5000/devfile-nodejs-deploy:0.1.0" // from Devfile yaml file
 						dockerfilePath := filepath.Join(commonVar.Context, "Dockerfile")
 						buildCtx := commonVar.Context
-						Expect(lines[nbLines-2]).To(BeEquivalentTo(
+						Expect(lines[nbLines-2]).To(ContainSubstring(
```

Comment:
I think checking an exact line number to match might be still a bit fragile, so I'd suggest checking that `stdout` contains the expected string:

```suggestion
						Expect(stdout).To(ContainSubstring(
```